src/tests/src/libs/antares/study/wrappers/study-wrapper.h

```diff
@@ -0,0 +1,79 @@
+
+#include <study.h>
+
```

Comment:
**study-wrapper.h** isn't part of any CMakeFile. It may result in an incomplete dependency graph.